libvirt/domain_test.go

```diff
@@ -0,0 +1,248 @@
+package libvirt
+
```

Comment:
I would rename `domain_test.go` with `guestfish_test.go` or something that doesn't impact a `domain` imho